plugins/dell/fu-plugin-dell.c

```diff
@@ -885,6 +882,13 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 		return FALSE;
 	}
 
+	/* capsules not supported */
+	if (!fu_plugin_dell_capsule_supported (plugin)) {
```

Comment:
the reason this is failing is that `data->capsule_supported` isn't populated until line 900.

Reply:
And as adjusting it, I think that line 901-903 probably can go away too.

Reply:
Gotcha, thanks!